common.json

```diff
@@ -19,6 +19,20 @@
 			"files": [ "**/*.yaml", "**/*.yml" ],
 			"parser": "yaml-eslint-parser",
 			"extends": "./yaml"
+		},
+		{
+			"files": [
+				"**/*.ts"
+			],
+			"extends": [
+				"./typescript.js"
+			],
+			"rules": {
+				"comma-dangle": [
```

Comment:
We can just put this rule in `typescript.js` - like we do with the other overrides (e.g. `json.json`)

Reply:
Done.